### ISSUE TITLE
Creating subscription results in error in orion-ld log

### ISSUE THREAD

**Issue by pdmangel2** [Fiware/context.Orion-LD#1317]
Following creation of subscription in version
```

###
POST http://localhost:1026/ngsi-ld/v1/subscriptions
Fiware-Service: t_02
Content-Type: application/json

{
    "id": "urn:subscr:123457",
    "type": "Subscription",
    "description": "manual test",
    "entities": [
        {
            "idPattern": ".*",
            "type": "ParkingSpot"
        }
    ],
    "q": "status==%22occupied%22",
    "watchedAttributes": ["status"],
    "notification": {
        "endpoint": {
            "uri": "http://nginx:8080/api/tools/mock/httpNoAuth",
            "accept": "application/ld+json"
        },
        "format": "keyValues",
        "attributes": [
            "status"
        ]
    }
}
```

invokes following error in orion-ld log:
`time=Thursday 02 Feb 10:25:32 2023.853Z | lvl=ERROR | corr=N/A | trans=N/A | from=N/A | srv=N/A | subsrv=N/A | comp=Orion | op=globals.cpp[544]:parse8601Time | msg=Not a valid date: '%22occupied%22'`

Replacing the `%22` by `\"` invokes in similar logging of `msg=Internal Error (the time string is too long - 41 characters is the max length)`

**Comment by kzangeli** [comment]
ok, noted.
I'll look into this as soon as I get an opportunity.

**Comment by kzangeli** [comment]
ok, understood.
The error message is unfortunate. Quite unfortunate!!!

The broker checks whether it's a valid timestamp and if it is, it parses it into a float.
As it is not a valid timestamp, it's understood as a normal string.

All OK, the subscription is created, just a quite annoying (and incorrect) error message in the logfile.
I'll remove it in a few minutes.

**Comment by kzangeli** [comment]
Fixed in PR #1318

**Comment by pdmangel2** [comment]
Fix tested and ok
